server/server.go

```diff
@@ -933,12 +935,9 @@ func getToken(md metadata.MD) string {
 
 	// check the HTTP cookie
 	for _, t := range md["grpcgateway-cookie"] {
```

Comment:
The `server.go` filtere cookies, should we do it here as well?

Reply:
I didn't want to add an additional loop here. I will add a filter logic inside `JoinCookies` loop.

Reply:
Agree. Since `JoinCookies` has filtering now we should remove it in `logout.go` as well https://github.com/argoproj/argo-cd/pull/5497/files#diff-49aa5cdf3cd8adb7bb4d8c1f6fe1cb5a0c665977142b7f33d183ebc4b0b0a1b8R69